src/sentry/eventstore/models.py

```diff
@@ -716,18 +716,18 @@ def __init__(
         data: NodeData,
         snuba_data: Mapping[str, Any] | None = None,
         occurrence: IssueOccurrence | None = None,
-    ):
+    ) -> None:
         super().__init__(project_id, event_id, snuba_data=snuba_data)
         self.group = group
         self.data = data
         self._occurrence = occurrence
 
-    def __eq__(self, other):
+    def __eq__(self, other: BaseEvent) -> bool:
         if not isinstance(other, GroupEvent):
             return False
         return other.event_id == self.event_id and other.group_id == self.group_id
 
-    def __hash__(self):
+    def __hash__(self) -> str:
```

Comment:
`__hash__` should always have the signature `def __hash__(self) -> int:`

makes me think this file isn't being checked properly 🤔